<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SurfaceBuilder</class>
 <widget class="QFrame" name="SurfaceBuilder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QFrame" name="horizontalFrame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonClear">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="selectClear">
        <item>
         <property name="text">
          <string>All</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Walls</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Spawns</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Teleporters</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Snakes</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="selectTool"/>
      </item>
      <item>
       <widget class="QPushButton" name="buttonLoad">
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSave">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="surfaceRows">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
